subsys/bootloader/Kconfig

```diff
@@ -12,13 +12,14 @@ config MCUBOOT_BUILD_S1_VARIANT
 	help
 	  Build upgrade candidate of MCUBoot for alternative slot S1.
 
-config SECURE_BOOT_CRYPTO_CLIENT
-	bool "Include Secure Bootloader crypto client API"
+config SECURE_BOOT_CLIENT_API
+	bool "Include Secure Bootloader client API"
+	select FW_METADATA
```

Comment:
This should be "depends on" 

From the zephyr docs: `Avoid selecting symbols with prompts or dependencies. Prefer depends on`

Reply:
Done